### ISSUE TITLE
Publish imgpkg to TanzuNet

### ISSUE THREAD

**Issue by StevenLocke** [carvel-dev/imgpkg#123]
Why?

We want people to be able to consume imgpkg off of TanzuNet. We'd like imgpkg available on TanzuNet alongside the other tools in the suite.

TODO:
- [x] Learn how the other tools are made available (check the index?)
- [x] Provide an `access_token` to the `github-releases` resource so we don't get rate limited in the future.
- [x] Add license file generation to all tools in concourse - maybe in [this norsk](https://github.com/pivotal-cf/norsk-config/tree/master/releases) repo
- [x] Merge the [imgpkg norsk PR](https://github.com/pivotal-cf/norsk-config/pull/1412)
- [x] Request a new product in OSM for imgpkg
- [x] Request a new product in OSM for other tools?
- [x] Merge [carvel tools norsk pr](https://github.com/pivotal-cf/norsk-config/pull/1421)
- [x] submit PR for other carvel tools to norsk-config (not just imgpkg)
  - [x] blocked on OSPO approval: [https://osm.eng.vmware.com/oss/#/version-cloning-requests/21425 https://osm.eng.vmware.com/oss/#/version-cloning-requests/21426 https://osm.eng.vmware.com/oss/#/version-cloning-requests/21427] 
- [x] [Create a mirror](https://github.com/pivotal-cf/norsk-config/wiki/Mirror-Norsk-Status-in-your-Concourse) of the norsk pipeline in our concourse
  - [x] configure norsk to use a public key for osco-scan https://github.com/pivotal-cf/norsk-config/pull/1419
  - [x] add deploy keys to oslo-scan github repo https://vmware.slack.com/archives/C0GKRPLE5/p1618865252116200
- [x] Create a 'new' product on TanzuNet "imgpkg"
  - [x] Blocked on waiting for tanzunet folk to create new product (sent email) (also reached out via slack https://vmware.slack.com/archives/C055G2ZGW/p1618250420229700)
- [x] Get OSL for each of the tools
- [x] Upload latest imgpkg to tanzunet (0.6.1) See https://network.pivotal.io/docs/faq#add_osl_file
- [x] Backfill OSL versions for other products latest version. See https://network.pivotal.io/docs/faq#add_osl_file
  - [x] blocked on tanzunet manually attaching osl file https://vmware.slack.com/archives/C055G2ZGW/p1619820428423000 
- [x] Update release process on [hackmd](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ)
- [x] discuss the osm/oslo process with the team during tech forum

Automation:
- [ ] Update k8slt pipieline when uploading binary to also upload OSL. see https://github.com/pivotal-cf/pivnet-resource/tree/master/metadata
- [ ] request a service account (for osm portal) using HelpNow+
use the template, name and reason for the service account.
- [ ] Automate snapshotting an OSM release See https://osm.eng.vmware.com/api/public/v1/docs/#release-create
- [ ] https://osm.eng.vmware.com/api/public/v1/docs/#release-update
request: release_from_id -> to clone 'Latest' product
- [ ] https://osm.eng.vmware.com/api/public/v1/docs/#release-osl_status
provide the release id to get the OSL id. 
https://osm.eng.vmware.com/api/public/v1/docs/#release-osl_download 
provide the OSL id for the OSL file

**Comment by StevenLocke** [comment]
The current system works via concourse at [hush-house](https://hush-house.pivotal.io/). We'll continue to use this system, adding in imgpkg, and further inquiring about whether OSL is needed.

Update: OSL is needed and is supposed to be version specific for each tool. We're working on getting norsk pipelines for the tools, such that we can provide their dependencies to OSM and they'll give us the OSL.

We requested a new OSM product for imgpkg, kapp, ytt, and kbld. Pending their approval before (we think) the PR for norsk can be merged.

**Comment by DennisDenuto** [comment]
I attached a package “imgpkg 0.6.0” to the OSM release “imgpkg 0.6.0"
And, I created a new product “imgpkg” with a version called “Latest” to use in our norsk pipeline.

The above 2 are pending OSPO approval.

**Comment by DennisDenuto** [comment]
Closing this issue (since imgpkg is published to tanzunet)

will create a new issue to capture the 'automation' piece to it.
